libs/coin-algorand/src/specs.ts

```diff
@@ -51,8 +64,10 @@ const pickSiblingsOptedIn = (siblings: Account[], assetId: string) => {
 // TODO: rework to perform _difference_ between
 // array of valid ASAs and array of ASAs currently
 // being opted-in by an account
-const getRandomAssetId = (account) => {
-  const optedInASA = account.subAccounts?.reduce((old, current) => {
+
+const getRandomAssetId = (account: Account): string | undefined => {
+  // FIXME: properly type reduce function and optedInASA
```

Comment:
Aren't `old` and `current` of type `SubAccount`?

Reply:
Not exactly (old is a `string`) but typing fixed here bd8d72e8cc7a19c698b28c5950a51733f65d491a